src/day8/re_frame/trace.cljs

```diff
@@ -145,31 +145,53 @@
     (fn [trace]
       (< (:query query) (:duration trace)))))
 
-(defn render-traces [showing-traces trace-detail-expansions]
+(defn add-filter [filter-items filter-input filter-type from-mouse?]
+  (swap! filter-items conj {:id (random-uuid)
+                            :query (if from-mouse?
```

Comment:
I think we can remove the from-mouse? parameter for now. That way we will always have the same behaviour for a certain filter type. We can then implement the exact keyword matching stuff as a new filter type (:op-type?) in #39.

Reply:
sounds good!